application/src/View/Helper/DataType.php

```diff
@@ -74,6 +74,9 @@ public function getSelect($name, $value = null, $attributes = [])
         $element->setEmptyOption('Default')
             ->setValueOptions($options)
             ->setAttributes($attributes);
+        if (!$element->getAttribute('multiple') && is_array($value)) {
```

Comment:
Is this stray stuff from the multiple datatypes change?

Reply:
It's separated : as long as there is a data type helper, it should manage a form with single and multiple data types, somewhere else than in the resource template form. But indeed, it is needed for the multiple data types.